src/cobra/flux_analysis/deletion.py

```diff
@@ -405,3 +405,47 @@ def double_gene_deletion(
         processes=processes,
         **kwargs
     )
+
+
+@pd.api.extensions.register_dataframe_accessor("knockout")
+@pd.api.extensions.register_series_accessor("knockout")
```

Comment:
Is there a use-case for `Series`? I think it's always a `DataFrame`, isn't it?

Reply:
In the tests it's always used as:

```
growth = results.growth
...
assert growth[rxn1] ...
```

This can only be handled if you have an accessor on the columns as well.